Examples/Tests/initial_distribution/analysis_distribution.py

```diff
@@ -96,17 +96,34 @@
 
 # parameters of theory
 x_rms = 0.25
+z_cut = 2.
 q_tot = -1.0e-20
 q_e   = -1.602176634e-19
 npart = q_tot/q_e
-db    = 0.04
+db    = bin_value[1]-bin_value[0]
 
 # compute the analytical solution
-f = npart*db * np.exp(-0.5*(bin_value/x_rms)**2)/(x_rms*np.sqrt(2.0*scc.pi))
-f_peak = np.amax(f)
+f_xy = npart*db * np.exp(-0.5*(bin_value/x_rms)**2)/(x_rms*np.sqrt(2.0*scc.pi)) * scs.erf(z_cut/np.sqrt(2.))
+f_z = npart*db * np.exp(-0.5*(bin_value/x_rms)**2)/(x_rms*np.sqrt(2.0*scc.pi))
+f_z[ np.absolute(bin_value) > z_cut * x_rms ] = 0.
+f_peak = np.amax(f_z)
 
 # compute error
-f4_error = np.sum(np.abs(f-h4x)+np.abs(f-h4y)+np.abs(f-h4z))/bin_value.size / f_peak
+f4_error = np.sum(np.abs(f_xy-h4x)+np.abs(f_xy-h4y)+np.abs(f_z-h4z))/bin_value.size / f_peak
+
+do_plot = False
+if do_plot:
```

Comment:
this is unreachable code by default, maybe add a cmd line arg?

Reply:
I agree this piece of code adds a warning, but I am not sure adding lines to make it a command-line argument and document it helps the code, if it's just to remove a warning... If you want to get rid of the warning, I am fine with just deleting the plotting option.

Reply:
Ha, you don't get out that easily ;-)
See:
```suggestion
def usage():
    print("usage:")
    print("  {} [-p,--plot]".format(sys.argv[0]))

import sys
import getopt        
try:
    opts, args = getopt.getopt(sys.argv[1:], "hp",
        ["help", "plot"])
except getopt.GetoptError as err:
    print(err)
    usage()
    sys.exit(2)

for opt, arg in opts:
    if opt in ["-h", "--help"]:
        usage()
        sys.exit()
    elif opt in ["-p", "--plot"]:
        do_plot = True

if do_plot:
```

Reply:
:'(